ospf6d/subdir.am

```diff
@@ -83,3 +83,6 @@ ospf6d_ospf6d_snmp_la_SOURCES = ospf6d/ospf6_snmp.c
 ospf6d_ospf6d_snmp_la_CFLAGS = $(WERROR) $(SNMP_CFLAGS) -std=gnu99
 ospf6d_ospf6d_snmp_la_LDFLAGS = -avoid-version -module -shared -export-dynamic
 ospf6d_ospf6d_snmp_la_LIBADD = lib/libfrrsnmp.la
+
+ospf6d/ospf6_asbr_clippy.c: $(CLIPPY_DEPS)
```

Comment:
Two issues here:
- Please use `clippy_scan += foo` instead of this.
- Missing newline at the end of line

Please try this instead:

```
clippy_scan += \
       ospf6d/ospf6_asbr.c \
       # end
```

Reply:
Fixed